src/utils/actions.js

```diff
@@ -1,5 +1,96 @@
-export const createActiontypes = actionTypes => actionTypes
+import React from "react";
+import { Avatar } from "react-native-elements";
+import { View, Text } from "react-native";
+
+// Created this functional component to fix current Avatar flickering issue on ios
+// Should be fixed in the next React Native Elements Update
+const userAvatar = ({ item }) => {
+	return (
+		<Avatar
+			size = "large"
+			rounded
+			source = {{ uri: item.picture }}
+		/>
+	);
+};
+export const StableAvatar = React.memo(userAvatar);
+
+export const createActionTypes = actionTypes => actionTypes
 	.map(type => ({ [type]: `module/${type}`	}))
-	.reduce((types, type) => ({ ...types, ...type }),
-		{}
-	);
\ No newline at end of file
+	.reduce((types, type) => ({ ...types, ...type }), {});
+
+// MemberPanel needs should be made into it;s own component
+export const MemberPanel = (user) => {
+	const {
+		textStyle,
+		contentContainerStyle,
+		userInfoContainer,
+		fullFlex,
+		AvatarContainer
+	} = styles;
+
+	return (
+		<View style = { contentContainerStyle }>
+			<View style = { userInfoContainer }>
+				<Text style = { [textStyle, fullFlex ] }>{ `${user.firstName} ${user.lastName}` }</Text>
+				<View style = { AvatarContainer }>
+					{ user.picture === ""
+					&& <Avatar
+						size = "large"
+						rounded
+						titleStyle = {{ backgroundColor: user.color }}
+						overlayContainerStyle = {{ backgroundColor: user.color }}
+						title = { user.firstName[0].concat(user.lastName[0]) }
+					/> }
+					{ user.picture !== ""
+					&& <StableAvatar
+						item = { user }
+					/> }
+				</View>
+			</View>
+		</View>
+	);
+};
+
+export const rankMembers = (sortedMembers, userId) => {
```

Comment:
Oh I commented about this earlier. This is good. But you need to delete the rank Members code from the dahsboard